alacritty/src/input/mod.rs

```diff
@@ -385,8 +387,13 @@ impl<T: EventListener> Execute<T> for Action {
             Action::CreateNewWindow => ctx.create_new_window(None),
             #[cfg(target_os = "macos")]
             Action::CreateNewTab => {
-                let tabbing_id = Some(ctx.window().tabbing_id());
-                ctx.create_new_window(tabbing_id);
+                // attatching new windows as tabs only work with decorations enabled
+                if ctx.config().window.decorations == Decorations::None {
+                    ctx.create_new_window(None);
+                } else {
+                    let tabbing_id = Some(ctx.window().tabbing_id());
+                    ctx.create_new_window(tabbing_id);
```

Comment:
Is there a reason to create a new window here? If the user wanted to, they could bind that, right?

Reply:
This code is same as the current code. I simply added the safeguard to not attempt to create the window with a tab id to avoid crashes.

Reply:
Yeah but why not no-op? The user didn't ask for a window.

Reply:
Now I understand, yes, if they asked for a tab, they shouldn't get a window. 

If they can't get a tab, the application shouldn't crash though.

Updated PR to reflect this.